tslint-config.json

```diff
@@ -83,9 +83,9 @@
     "no-submodule-imports": false,
     "no-switch-case-fall-through": true,
     "no-this-assignment": true,
-    "no-unbound-method": true,
-    "no-unnecessary-class": true,
-    "no-unsafe-any": true,
+    "no-unbound-method": [true, "ignore-static"],
+    "no-unnecessary-class": [true, "allow-empty-class", "allow-empty-constructor"],
+    "no-unsafe-any": false,
```

Comment:
Too bad we have to remove this rule. It's an important one.

Reply:
I agree! But if an external module uses the any type, this rule always raises an error 😕

Reply:
I think I got around this rule by casting the parameters, but I did not have to do it a lot of times.

Reply:
Well, with decorators, it is pretty annoying. For instance:

```ts
@ApiOkResponse({
    description: 'Successful operation',
  })
  @ApiBadRequestResponse({
    description: 'Bad request',
  })
  @ApiNotFoundResponse({
    description: 'Not found',
  })
  public async find(@Param('folderId', ObjectIDValidationPipe) folderId: string): Promise<Document[]> {
    return this.documentsService.find(folderId);
  }
```

Each decorator return any type, how can we cast properly these return types?

Reply:
We can't cast decorators. The only solution I found is to overwrite the typings of the modules. 😢

Reply:
Is it possible to override in each project this field instead of setting it to `false` for all of them? I think it's better

Reply:
Yes indeed, we can do this!